src/connect.js

```diff
@@ -6,7 +6,7 @@ var ethrpc = require("ethrpc");
 var contracts = require("./contracts");
 var api = require("./api");
 var rpcInterface = require("./rpc-interface");
-var connectToAugurNode = require("./augur-node").connect;
+var augurNode = require("./augur-node").connect;
```

Comment:
I think this needs to be just `var augurNode = require("./augur-node");` since you reference `augurNode.connect` below.